### ISSUE TITLE
[Debian] Which package to install the LSP plugin?

### ISSUE THREAD

**Issue by chatchoi** [geany/geany#4177]
I followed the instruction on https://nightly.geany.org/debian/. My Debian is set to not install recommended packages. When I install Geany, it will only install `geany` and `geany-common`.

**Comment by chatchoi** [comment]
I installed the `geany-plugins` package. It installs all of the plugins, but the LSP Client plugin is not there.

**Comment by techee** [comment]
> I installed the geany-plugins package. It installs all of the plugins, but the LSP Client plugin is not there.

Correct, the LSP plugin hasn't been merged into geany-plugins yet, see https://github.com/geany/geany-plugins/pull/1331. Even after that, it will take some time to get packaged by distributions.

In the meantime, I suggest compiling and installing it by yourself.
